src/polar/PolarRadiusAxis.tsx

```diff
@@ -9,11 +9,6 @@ import Layer from '../container/Layer';
 import { polarToCartesian } from '../util/PolarUtils';
 import { PresentationAttributes, filterProps, BaseAxisProps, TickItem, adaptEventsOfChild } from '../util/types';
 
-interface TickIem {
```

Comment:
Unused interface. The correct version `TickItem` is already being imported, so can remove this one.